scss/components/_case-custom-fields.scss

```diff
@@ -1,12 +1,9 @@
 .civicase__case-custom-fields__container {
-  float: left;
-  width: 50%;
-
   &.civicase__summary-tab-tile {
     padding-top: 0;
   }
 
-  .civicase__panel-transparent-header:not(:first-child) {
+  [civicase-masonry-grid-item]:nth-child(1n+2) {
```

Comment:
this looks pretty ad-hoc, what is it for? why isn't it in `_masonry-grid.scss`?

Reply:
This is a margin that is added to the 2nd element onwards.

<table>
  <tr>
    <td>1</td><td>2</td>
  </tr>
  <tr>
    <td><br>3</td><td><br>4</td>
  </tr>
  <tr>
    <td><br>5</td><td></td>
  </tr>
</table>

This is not done on the grid component because the margin could be different elsewhere.